src/_pytest/_code/code.py

```diff
@@ -43,7 +43,7 @@
 
     from _pytest._code import Source
 
-    _TracebackStyle = Literal["long", "short", "line", "no", "native"]
+    _TracebackStyle = Literal["long", "short", "line", "no", "native", "no_pytrace"]
```

Comment:
Could you please explain why we need `no_pytrace`?

Reply:
as far as i understood `pytrace=False` now uses a `ExceptionInfo`

Reply:
I 've tried to have the same output like in the previously version with ``str`` - that version used only ``excinfo.value`` (see https://github.com/pytest-dev/pytest/blob/33128200519e51a7d3aa8d2186ea6927f6a999f4/src/_pytest/nodes.py#L325 ). So my proposition is creating ``ReprExceptionInfo`` also with only ``excinfo.value``. I've tested all previously existed styles and all of them contains in ``ReprExceptionInfo`` information which are not needed in output.

Reply:
Using new style is definitely not the only solution. We may use another style (for example "no") and accept a little different output.

Reply:
I took a better look and now I understand things better, thanks.

I think the implementation is on point, perhaps the name `"no_pytrace"` could be bikeshed a bit though.

* `"no"`: it is an option, although one can argue that we are showing at least the exception message, so it is not really "no" style.

* `"value"`: we are showing just the message of the exception value

* `"line"`: showing just one line of the exception

Any other suggestions?

Reply:
``"no"`` and ``"line"`` styles already exist.

Reply:
Oh duh, sorry about that.

Yeah `"value"` seems good then. 👍 

@RonnyPfannschmidt want to chime in?

Reply:
I like value

Reply:
Cool.

@CarycaKatarzyna would you mind changing to `value` then? Thanks!

After that I think this is good to merge.

Reply:
Done.